roscala/src/main/scala/coop/rchain/rosette/VirtualMachine.scala

```diff
@@ -829,4 +853,11 @@ object VirtualMachine {
 
   def execute(op: OpUnknown, state: VMState): VMState =
     state.set(_ >> 'exitFlag)(true).set(_ >> 'exitCode)(1)
+
+  def inspect[A](f: VMState => A): State[VMState, A] =
+    State.inspect[VMState, A](f)
```

Comment:
Is there any reason to not use a curried form? 
`val inspect = State.inspect[String,Int] _`